File: Iniciante/1064 - Positivos e Média/Python/1064.py
def main():
    QUANTITY_OF_VALUES = 6
    count = 0
    total = 0.0

    for _ in range(QUANTITY_OF_VALUES):
        value = float(input())
        if value > 0:
            count += 1
            total += value

    print(f"{count} valores positivos")
    print(f"{total / count:.1f}")


if __name__ == "__main__":
    main()

File: Iniciante/1064 - Positivos e Média/JavaScript/1064.js
function main() {
    const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
    const QUANTITY_OF_VALUES = 6;
    let count = 0;
    let sum = 0.0;

    for (let i = 0; i < QUANTITY_OF_VALUES; i++) {
        const value = parseFloat(input[i]);
        if (value > 0) {
            count++;
            sum += value;
        }
    }

    console.log(`${count} valores positivos`);
    console.log((sum / count).toFixed(1));
}

main();
